### ISSUE TITLE
Support for 2fa

### ISSUE THREAD

**Issue by zugzwang** [mugartec/CLAP#1]
Great project, thanks.

I would like to propose support for 2fa with OTP tokens, using `oathtool`. Users can assign OTP generators to already CLAPped passwords.

Suggested flow:

```
$ clap -g myNewPassword
      [create, encrypt password]
$ clap --add-token myNewPassword
      Insert master password: <master password>
      Insert TOTP secret: <input otp key>
$ clap -c myNewPassword
      Insert master password: <master password>
      [copies password to clipboard]
      Press Enter to obtain OTP token (Ctrl+C to cancel): <Enter>
      [copies OTP token to clipboard]
```

**Comment by mugartec** [comment]
Thanks @zugzwang for the suggestion.

The flow looks great, please feel free to submit a PR.

**Comment by mugartec** [comment]
Closed by #2.
